databricks_cli/runs/cli.py

```diff
@@ -106,12 +112,32 @@ def list_cli(api_client, job_id, active_only, completed_only, offset, limit, out
       - Result state (can be n/a)
     """
     check_version(api_client, version)
-    runs_json = RunsApi(api_client).list_runs(
-        job_id, active_only, completed_only, offset, limit, version=version)
+    api_version = version or api_client.jobs_api_version
+    if api_version != '2.1' and (expand_tasks or offset or limit or _all):
+        click.echo(click.style('ERROR', fg='red') + ': the options --expand-tasks, ' +
+                   '--offset, --limit, and --all are only available in API 2.1', err=True)
+        return
+    runs_api = RunsApi(api_client)
+
+    has_more = True
+    runs = []
+    if _all:
+        offset = 0
+        limit = 20
+    while has_more:
```

Comment:
Should we make this more generic since `jobs/list` does the same thing.

Reply:
@nordp Do you want to address this here or in a follow up PR?

Reply:
That's fine. Can you rebase?